boot/freeldr/freeldr/arch/i386/pc/pchw.c

```diff
@@ -288,9 +289,9 @@ DetectBiosFloppyController(PCONFIGURATION_COMPONENT_DATA BusKey)
         ERR("Failed to allocate resource descriptor\n");
         return NULL;
     }
-    memset(PartialResourceList, 0, Size);
 
     /* Initialize resource descriptor */
+    memset(PartialResourceList, 0, Size);
```

Comment:
Please use RtlZeroMemory consistently

Reply:
Agreed. I add commits 2 and 3, which I'll merge after approval.